### DISCUSSION TITLE
Blazor Hybrid aka Dotnet Maui Build/Publishing Issue with iOS Target

### DISCUSSION THREAD

**Discussion by AathifMahir** [dotnet/maui#19948] [Q&A]
Hello Folks,

Anyone has a Idea on Resolving Below Build Issues when Publishing Blazor Hybrid aka Dotnet Maui App?

wwwroot/favicon.ico : error : The path '../../../../../../src/ProjectName/wwwroot/favicon.ico' would result in a file outside of the app bundle and cannot be used. 

wwwroot/index.html : error : The path '../../../../../../src/ProjectName/wwwroot/index.html' would result in a file outside of the app bundle and cannot be used.

appsettings.json : error : The path '../../../../../../src/ProjectName/appsettings.json' would result in a file outside of the app bundle and cannot be used.

This is first building Native Project that I had for Maui App which doesn't contain any Razor Components, I assume I would face the same on Shared Razor Class Library that contains all the Razor Components.

**Comment by AathifMahir** [comment]
I Assume that I have Solved favicon.ico and index.html issues but the appsettings.json is biggest problem here,

appsettings.json : error : The path '../../../../../../src/ProjectName/appsettings.json' would result in a file outside of the app bundle and cannot be used.

Appriciate if anyone can hop in and help with this, By the way I already went through Most of the issues related this and even tried the examples in below url

https://github.com/xamarin/xamarin-macios/blob/main/dotnet/BundleContents.md

This is how the json files were added in csproj

```xml
<ItemGroup>
	<EmbeddedResource Condition="'$(Configuration)' == 'Release'" Include="appsettings.json"/>
	<EmbeddedResource Condition="'$(Configuration)' == 'Debug'" Include="appsettings.development.json"/>
</ItemGroup>
```

**Comment by Marini88** [comment]
how did you solve it?

**Comment by riaanswart** [comment]
I have this issue with all my files that is in a razor shared library.  Do you have any information of how you fixed this? Trying to publish the ios app but not able to

**Comment by Skarzag** [comment]
Exact same problem with EmbeddedResources with azure pipeline (work on my computer)

**Comment by riaanswart** [comment]
Ok so as soon as I upgraded the projects to .net 9 and all package updates got the publish running on azure devops, it did timeout but disabling LLVM helped
/p:MtouchUseLlvm=false
